index.html

```diff
@@ -40,7 +40,7 @@
       }
 
       #output-div {
-        background-color: lightgrey;
+        background-color: rgb(254, 255, 243);
```

Comment:
It's great that you're playing with rgb for the display! This adds a great deal of personal touch to your code!